### ISSUE TITLE
More events

### ISSUE THREAD

**Issue by KarlOfDuty** [northwood-studios/NwPluginAPI#57]
106 events:
- [ ] PlayerEnterFemurBreaker
- [ ] PlayerContain106 (Specifically when someone presses the button)
- [ ] Player106Teleport
- [ ] Player106CreatePortal
- [ ] PlayerPocketDimensionEnter
- [ ] PlayerPocketDimensionLeave
- [ ] PlayerPocketDimensionDie (?)

Not sure about the last one, we had it in smod but I can't really see how it is useful in comparison to the normal death event.

- [ ] PlayerSetNickname(Player player, string oldName)
- [ ] PlayerThrowGrenade - Preferably allowing the plugin to change things like direction and throwing power
- [ ] PlayerInfect - Things like amount of damage and time they will be infected would be useful.

**Comment by KarlOfDuty** [comment]
Yep

**Comment by zabszk** [comment]
6 of the requested events have been introduced in #69.

**Comment by KarlOfDuty** [comment]
@zabszk I can't really find any 106 portal events, what exactly are they called?

**Comment by zabszk** [comment]
> I can't really find any 106 portal events, what exactly are they called?

Portals don't exist in 12.0.0.

**Comment by KarlOfDuty** [comment]
Ah I see

**Comment by KarlOfDuty** [comment]
Can 106 teleport in any way then?

**Comment by zabszk** [comment]
> Can 106 teleport in any way then?

Nope. It has been replaced with the stalk mode.

**Comment by KarlOfDuty** [comment]
Added on some more events btw

**Comment by SrLicht** [comment]
What is PlayerInfect?

**Comment by KarlOfDuty** [comment]
> What is PlayerInfect?

That's when 049 attacks someone so they get infected for a short time and if they die during it 049 can revive them.

If that is still how the game mechanic works.

**Comment by SrLicht** [comment]
But you can track the same using PlayerDamage

**Comment by SrLicht** [comment]
the damage per tick is done by the heart attack effect

**Comment by KarlOfDuty** [comment]
This one would have info on how long the infection lasts as well as the other damage information.

**Comment by KarlOfDuty** [comment]
Right, I was thinking more about the amount of time they would be revivable if they die within it.
